barretenberg/cpp/src/barretenberg/vm/tests/helpers.test.cpp

```diff
@@ -239,4 +239,13 @@ void clear_range_check_counters(std::vector<Row>& trace, uint256_t previous_valu
     trace.at(lookup_value + 1).lookup_u16_14_counts = trace.at(lookup_value + 1).lookup_u16_14_counts - 1;
     previous_value >>= 16;
 }
+
+// Need to use this instead of std::format("{:08x}") because it is not available in gcc builds
+std::string to_hex(uint32_t value)
```

Comment:
Was this routine copied from avm_opcode.cpp? Then, can we have a single instance of it?